luminance/src/render_state.rs

```diff
@@ -28,13 +28,27 @@ impl RenderState {
     B: Into<Option<Blending>>,
   {
     RenderState {
-      blending: blending.into(),
+      blending: blending.into().map(|x| x.into()),
+      ..self
+    }
+  }
+  /// Override the blending configuration using separate blending.
+  pub fn set_blending_separate<B1, B2>(self, blending_rgb: B1, blending_alpha: B2) -> Self
+  where
+    B1: Into<Blending>,
+    B2: Into<Blending>,
```

Comment:
Why do you use `Into<Blending>` instead of `Blending` here? In the case of `RenderState::set_blending`, the `Into<Option<Blending>>` allows to pass `blending` instead of `Some(blending)`, but here, I don’t get the advantage.